docs/notification_system.md

```diff
@@ -11,6 +11,40 @@ At a high level, the typical notification task flow is:
 * User creates a `NotificationTemplate` at `/api/v2/notification_templates/`.
 * User assigns the notification to any of the various objects that support it (all variants of Job Templates as well as organizations and projects) and at the appropriate trigger level for which they want the notification (error, success, or any). For example, a user may wish to assign a particular Notification Template to trigger when `Job Template 1` fails.
 
+## Templated notification messages
+
+When creating a notification template, the user can optionally provide their own custom messages for each notification event (start, success, error). If a message is not provided, the default message generated by Tower will be used.
+
+The notification message can include templated fields written using Jinja templates. The templates may reference any information (TBD) found in the associated job's serialization.
```

Comment:
> The templates may reference any information (TBD) found in the associated job's serialization.

What's the status of the TBD here?

Reply:
Good catch - updated. Only white-listed fields can be referenced in a job message.